### PR TITLE
[PHEE-699] Parallel Payee-Multi-Tenancy-Support in channel Transfer API

### PR DESCRIPTION
**Pull request by Anover000**
[PHEE-699 Parallel Payee-Multi-Tenancy-Support in channel Transfer API](https://fynarfin.atlassian.net/browse/PHEE-699?atlOrigin=eyJpIjoiZmUzYzljYjFlNDE1NDg1MWEwNTJhM2YxZDg1NzgwMWIiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!
- [ ] Followed the PR title naming convention mentioned above.

- [ ] Acknowledge that we will not merge PRs that are not passing the checks ("green") - it is your (author's) responsibility to get a proposed PR to pass all the checks, not primarily the project's maintainers.

- [ ] The PR title should include a JIRA ticket

- [ ] Design-related bullet points or design document links related to this PR added in the description above.

- [ ] Updated corresponding Postman Collection or API documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

- [ ] Followed coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

FYI our guidelines for code reviews same as https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Review by fynmanoj — APPROVED** [review]

File: src/test/java/resources/mojaloop.feature [hunk 1]
```diff
@@ -77,7 +78,8 @@ Feature: Mojaloop test
     Then I call the deposit account endpoint with command "deposit" for amount 10 for "payee"
 
     Then I add "payer" to als
-    Then I add "payee" to als
+#    payee/payee2/payee3
```
Comment (fynmanoj) [inline src/test/java/resources/mojaloop.feature]: why payee is comented? are we doing this test only for payee3 ?
can you add this test description accordingly?

**Review by fynmanoj — CHANGES_REQUESTED** [review]
need more clarity on the test case name. 
and the intended use case

File: src/test/java/resources/mojaloop.feature (hunk 1 above)
Comment (Anover000) [reply src/test/java/resources/mojaloop.feature]: The test can be done for any of the payees, that's what the comment intends to tell, this test does not intend to test Multi-tenancy. But these changes help people refer to the test cases and use the same step definitions used in this test.

Comment (Anover000) [reply src/test/java/resources/mojaloop.feature]: Should I make the comments a bit more clear?

Comment (fynmanoj) [reply src/test/java/resources/mojaloop.feature]: suggestion is to keep it in a data table so that when needed these can be run for multiple payees.

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: src/test/java/resources/mojaloop.feature (hunk 1 above)
Comment (Anover000) [reply src/test/java/resources/mojaloop.feature]: Scenario Outline was the right approach to implement your suggestion. I have updated the test.

Comment (Anover000) [reply src/test/java/resources/mojaloop.feature]: image

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**